APMrover2/Steering.cpp

```diff
@@ -311,7 +311,11 @@ void Rover::set_servos(void) {
         if (!apply_skid_mix) {  // Mixage is already done by a controller so just pass the value to motor
             motor1 = steering_scaled;
             motor2 = throttle_scaled;
+        } else if (channel_throttle->in_trim_dz()) {  // Use full range for on spot turn
```

Comment:
well, this is blocking in all auto mode ... cause channel_throttle->in_trim_dz() is input from radio ...